core/modules/node/node.pages.inc

```diff
@@ -458,6 +513,200 @@ function node_form_submit($form, &$form_state) {
   }
 }
 
+/**
+ * Generates a node preview.
+ *
+ * @param $node
+ *   The node to preview.
+ *
+ * @return
+ *   An HTML-formatted string of a node preview.
+ *
+ * @see node_form_build_preview()
+ */
```

Comment:
params in docblock should match the function.

Reply:
Done